src/Universe.h

```diff
@@ -83,7 +83,7 @@ class Universe {
 
 public:
 
```

Comment:
Thanks for picking up on that @samuelshaner, the latest commit makes the ``Lattice`` default ID -1 as well.